index.html

```diff
@@ -296,13 +296,13 @@ <h3>
           <li>The <dfn id='dom-PerformanceNavigationTiming-workerStart'><code>
             workerStart</code></dfn> attribute MUST return the time immediately
             before the user agent <a href=
-            "https://www.w3.org/TR/service-workers/#service-worker-concept">ran
+            "https://www.w3.org/TR/service-workers/#run-service-worker">ran
             the worker</a> (if the <a>current document</a> has an <a href=
             "https://www.w3.org/TR/service-workers/#dfn-containing-service-worker-registration">
-            active service worker registration</a> [[!SERVICE-WORKERS]])
+            active service worker registration</a> [[SERVICE-WORKERS]])
             required to service the request, or if the worker was already
             available, the time immediately before the user agent <a href=
-            "https://www.w3.org/TR/service-workers/#on-fetch-request-algorithm">
+            "https://www.w3.org/TR/service-workers/#fetchevent">
```

Comment:
as above.

Reply:
There are lot of these... go please fix globally.

Reply:
fixed everything I could find